node/malus/src/variants/suggest_garbage_candidate.rs

```diff
@@ -241,8 +280,30 @@ where
 	}
 }
 
+#[derive(Clone, Debug, clap::Parser)]
+#[clap(rename_all = "kebab-case")]
+#[allow(missing_docs)]
+pub struct SuggestGarbageCandidateOptions {
+	/// Determines the percentage of candidates that should be disputed. Allows for fine-tuning
+	/// the intensity of the behavior of the malicious node. Value must be in the range 0..=100.
+	#[clap(short, long, ignore_case = true, default_value_t = 100, value_parser = clap::value_parser!(u8).range(0..=100))]
+	pub percentage: u8,
```

Comment:
Why do we use `u8` and not `f64`/`f32` here? Percentages could be like `42.2` and it is totally legit I think. It will also be simplier to use them afterwards.

Reply:
Agreed

Reply:
I don’t think we need this kind of precision, but don’t have a strong preference here.

Reply:
The problem is that `u8` is not expressive here. It does not internally have a range like `0..100` nor has a direct conversion to `f64`.